framework/src/st_things/Make.defs

```diff
@@ -39,17 +38,12 @@ CFLAGS += -I$(THINGS_STACK_DIR)/src/common/easy-setup/inc
 CFLAGS += -I$(THINGS_STACK_DIR)/src/common/framework/inc
 CFLAGS += -I$(THINGS_STACK_DIR)/src/common/logging/inc
 CFLAGS += -I$(THINGS_STACK_DIR)/src/common/memory/inc
-CFLAGS += -I$(THINGS_STACK_DIR)/src/common/security/inc
 CFLAGS += -I$(THINGS_STACK_DIR)/src/common/utils/inc
-CFLAGS += -I$(THINGS_STACK_DIR)/src/common/stack
 CFLAGS += -I$(THINGS_STACK_DIR)/inc
-CFLAGS += -I$(THINGS_STACK_DIR)/src/common/security/ssCrypto/include
```

Comment:
Let's remain this and remove relative path.

Reply:
agree. we should remove #include starting with ".."
e.g) "../security/ssCrypto/include/ss_sha2.h"